### ISSUE TITLE
It doesn't reflect document changes

### ISSUE THREAD

**Issue by JC3** [nea/MarkdownViewerPlusPlus#147]
## Issue description

When I edit the Markdown file, the viewer doesn't update to show the changes. Pressing Refresh doesn't update it either. The only way to update the preview is to close the viewer pane then reopen it from the menu.

### Screenshot

![aa](https://user-images.githubusercontent.com/5651184/144459852-02039936-790d-4e92-bf5e-66a5dfe3c836.gif)

### Environment
* Notepad++ 8.1.9.2 x64
* MarkdownViewer++ 0.8.2
* Windows 10 x64

**Comment by ProfiSketch** [comment]
The same issue prevented the plugin from being used.

**Comment by marcuslehr** [comment]
Great screen capture. I'm having the same issue and it makes the plugin nearly unusable. I would hope this would update in real time. At the very least the refresh feature should be working and there should be a keyboard shortcut mapped to it.

**Comment by MrMartian** [comment]
Same here. This has been the case for me since the update to Notepad++ Version 8. Before that synchronisation worked well.
Using Windows 10 Pro along with Notepad++ Version 8.2.1.

**Comment by leitwolf7** [comment]
Interesting: I have 2 systems, one has MarkdownViewerPlus installed for several years now (updates applied when available) and on this system the immediate updates work well. 
Then I have another system where I installed MarkdownViewerPlus just a few days ago (also using the plugins admin functionality) and with this installation I see the bug that is described here. Both systems are on the same version level of both Notepad++ and the plugin. 
I have no idea why it works on the older installation.... maybe these additional info helps in some way.

**Comment by saephraim** [comment]
Same here, click View > Refresh after a change and nothing happens.

**Comment by daniu-de** [comment]
hmmm - yeah ... I do have the same issue. However I would appreciate if it would go one step further and just refresh automatically. Best would be synchronous with typing, every x second would also be OK

**Comment by MrMartian** [comment]
@daniu-de  - agreed, this would be by far the best solution, and as far as I remember it used to be like that, didn't it? Would be really great if this would work again. At the moment using Markdown in NP++ without a proper Preview is really cumbersome and not much fun.

**Comment by daniu-de** [comment]
well ... sure - however I would never do it the way in the above video: just use the icons

**Comment by JC3** [comment]
> well ... sure - however I would never do it the way in the above video: just use the icons

Fwiw the video was just meant to illustrate the problem, rather than to be instructional, heh.

I just use an online Markdown editor, personally. I'm still waiting for a functional one so I can do it right in Notepad++ but for now, fingers crossed. 

I'm assuming some Notepad++ API change broke this (and a few other plugins too) but, I don't really have the knowledge or motivation to dig into it myself. I wonder if somebody fixed it on a fork; I haven't looked through them tbh.

**Comment by PAW-Blackrock** [comment]
Still having the same issue as described above.  Makes the plugin not so useful, unfortunately.

N++ v8.4
M++ plugin v.0.8.2.25564

**Comment by daniu-de** [comment]
I am now using Notepad v8.3.2 with Plugin Version 0.8.2.25525 and do not have the issue. Works flawless

**Comment by jrschueller** [comment]
I'm having the same issue as described here.

Notepad++ v8.3.3
MarkdownViewer++ v0.8.25564
Windows 10 Enterprise x64 20H2 build 19042.1586
.NET Framework 4.8

**Comment by DavidBruchmann** [comment]
I can see changes by just switching to another tab and back.  
That is actually the same amount of clicks I had to do with the refresh button, so it's just another way without more efforts.

**Comment by jason-murdick-ub** [comment]
Confirmed, even on Notepad++ 8.4.6. Switching to another document or closing and reopening the viewer window refreshes it.  However, it does not auto-update and the Refresh command does not work.

**Comment by vlab22** [comment]
> I can see changes by just switching to another tab and back. That is actually the same amount of clicks I had to do with the refresh button, so it's just another way without more efforts.

Same problem here.

```Notepad++ v8.4.8   (64-bit)
OS Name : Windows 11 (64-bit) 
OS Version : 22H2
OS Build : 22621.1413
```

**Comment by NablaDeltaPsi** [comment]
Same here
Windows 10 (64 bit)
Notepad++ v7.8.8

**Comment by delscorcho** [comment]
Same issue, Notepad++ v8.6.2

**Comment by PifPof73** [comment]
Same issue, Notepad++ v8.6.5, 64bit

**Comment by kochul2000** [comment]
same issue, v8.6
I have to ( Ctrl + Shift + m ) * 2 to refresh

**Comment by giorgitsiklauri** [comment]
2024.. this problem remains actual. Is this plugin still maintained after all?

**Comment by giorgitsiklauri** [comment]
It's 2024.. and this problem remains actual. Is this plugin still maintained after all?

**Comment by JC3** [comment]
I don't think this plugin is maintained and I'm starting to wonder if there's some way to request NPP to remove it from the available plugin list.

**Comment by kingram6865** [comment]
I guess it needs to be removed. I just entered this as a new issue before checking to see if it was already entered.
4 years and still not fixed?
